java_script.js

```diff
@@ -195,4 +195,18 @@ for (let i = 0; i < popupBtns.length; i += 1) {
       document.body.removeChild(myWork);
     });
   });
-}
\ No newline at end of file
+}
+
+const contactFrom = document.querySelector('.form-elements');
+const [email] = contactFrom.elements;
+const validationError = document.querySelector('small');
+
+contactFrom.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const regex = /[A-Z]/;
+  if (!regex.test(email.value)) {
+    contactFrom.submit();
```

Comment:
- [ ] Kindly 😃 make sure that the functionality is working because when I write an email with a capital letter it is working and redirect me to the form spree page which means the functionality of the checking 💯 the email is in lowercase not working, so please try to fix that to let the function work and with that no one can enter invalid email and just the valid email will communicate with you 😊

> Check below to see what I am talking about 😊⤵️
> If you can not see the GIF image clearly just click on it and it will be opened in a new tab of the browser😊 

| Your form|  My form| 
| --- | --- |
| ![Recording 2023-02-04 at 16 52 34](https://user-images.githubusercontent.com/83360636/216774006-9d78c777-8a01-4164-a22f-7d25ae9679b0.gif) |  ![Recording 2023-02-04 at 16 54 53 (1)](https://user-images.githubusercontent.com/83360636/216774136-b33aeee3-770e-4f71-aa6e-0cb85d69babf.gif)  |